lib/pghero/methods/kill.rb

```diff
@@ -2,7 +2,8 @@ module PgHero
   module Methods
     module Kill
       def kill(pid)
-        execute("SELECT pg_terminate_backend(#{pid.to_i})").first["pg_terminate_backend"] == "t"
+        result = execute("SELECT pg_terminate_backend(#{pid.to_i})").first["pg_terminate_backend"]
+        truthy? result
```

Comment:
I found that having it all on one line made it harder to read (particularly the one with the HEREDOC), but if that's the style you prefer, I'll make the change.